.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+## How to contribute
+
+Share your knowledge with the community by contributing to Magento DevDocs!
+You can contribute by creating an issue or pull request (PR) on our [devdocs](https://GitHub.com/magento/devdocs) GitHub repository.
+We welcome all types of contributions; from minor typo fixes to new topics.
+
+Magento's tech writer team and [Maintainers](https://devdocs.magento.com/contributor-guide/devdocs-maintainers.html) review issues and pull requests on a regular basis, and we do our best to address all issues as soon as possible.
```

Comment:
"Community Maintainers" instead of just "Maintainers"?

Reply:
I dig it, adding